dangerzone/conversion/errors.py

```diff
@@ -0,0 +1,78 @@
+from typing import List, Optional, Type
+
+
+class ConversionException(Exception):
+    error_message = "Unspecificed error"
+    error_code = -1
+
+    def __init__(self, error_message: Optional[str] = None) -> None:
+        if error_message:
+            self.error_message = error_message
+        super().__init__(self.error_message)
```

Comment:
The reason why I didn't do so earlier was because I cannot see a clean way of doing it. Because we're talking about class attributes and not instance ones, we can't simply access other attributes like `self.error_code`. And combining `@property` with `@classmethod` [according to some was deprecated in python 3.11](https://stackoverflow.com/questions/128573/using-property-on-classmethods).

I've attempted a simple approach at 628cadf9. Let me know what you think.

Reply:
Oh, I was actually thinking of initializing those error classes in `exception_from_error_code`, and then matching the class with the exit code. What you suggest though is simpler, so let's go with that.